src/components/Modal/AletheiaModal.style.tsx

```diff
@@ -57,7 +57,7 @@ const AletheiaModal = styled(Modal)`
     }
 
     .hide-modal {
-        color: #db9f0d;
+        color: ${colors.warning};
```

Comment:
suggestion: We have two identical color with different names, we should pick only one to use across the platform

`lightYellow` and `warning`